apps/cowswap-frontend/src/common/updaters/FinalizeTxUpdater.tsx

```diff
@@ -175,11 +176,17 @@ function finalizeOnChainCancellation(
   orderId: string,
   sellTokenSymbol: string
 ) {
-  const { chainId, isSafeWallet, dispatch, addPopup, cancelOrdersBatch } = params
+  const { chainId, isSafeWallet, dispatch, addPopup, cancelOrdersBatch, account } = params
```

Comment:
![image](https://github.com/cowprotocol/cowswap/assets/43217/33e0a88d-7239-4efc-ada5-36945b396f83)

Addressed in a follow PR I assume

Reply:
Yep